### PR TITLE
use `mtt` action for integration tests

### PR DESCRIPTION
**Pull request by BuckarooBanzay**
This PR removes the manual integration-testing files and switches to the `mtt` action.

It currently tests against latest engine release `5.8.0` and current master/main branches of the dependencies (game and mods)

**Note:** The test against 5.5.0 is removed now due to some gh-action limitations (https://github.com/BuckarooBanzay/mtt/issues/4) IMO: we should let go of the old engine versions anyway

### CODE REVIEW

**Review by S-S-X — APPROVED** [review]
I'd be happier if test specific stuff would be in some sub dirs dedicated for tests, I mean similar how it was.

Anyway a lot cleaner than before overall and seems like just two files anyway.

**Comment by BuckarooBanzay** [comment]
> I'd be happier if test specific stuff would be in some sub dirs dedicated for tests, I mean similar how it was.

There's only the `mtt.lua` and `registered_nodes.txt` in the "main" technic mod-folder, i could move them to a "test" directory if you want...

**Comment by OgelGames** [comment]
I think it would be good to keep the `.test` directory for those files, even though it's only two; it keeps the mod directories cleaner, and they automatically get excluded from releases.

File: .github/workflows/test.yml
Comment (OgelGames) [inline .github/workflows/test.yml:1]: Do you think it would be better to name this `mtt`? The other workflows are named after their actions.

Comment (BuckarooBanzay) [reply .github/workflows/test.yml:1]: sounds reasonable, done :+1:

**Pull request merged**